Source/DafnyCore/Compilers/Compiler-python.cs

```diff
@@ -1096,6 +1107,8 @@ private class ClassWriter : IClassWriter {
       }
       if (dtv.Ctor.EnclosingDatatype is not TupleTypeDecl) {
         wr.Write($"{DtCtorDeclarationName(dtv.Ctor)}");
+      } else if (dtv.Ctor.Destructors.Count(d => !d.IsGhost) == 1) {
+        arguments += ",";
```

Comment:
This feels like the wrong place to handle commas, and confused as heck about what this is trying to do. Won't you end up with `([...],)` below?

Reply:
Which is exactly what you want, as the no comma version doesn't create a tuple.

Reply:
It's something we need to deal with as we use native tuples.

Reply:
RIGHT, please do add a comment - it looks so weird it feels like a typo. :)